public/form/resetpassword.html

```diff
@@ -1,5 +1,6 @@
 <html lang="en">
 <head>
+	<meta charset="utf-8">
```

Comment:
in the future, let's avoid changes that are not part of the current task. this may seem harmless, but this is how magical regressions get introduced